packages/core/src/constants.ts

```diff
@@ -469,3 +469,11 @@ export const RELATIVE_DATES = {
 } as const
 
 export type RelativeDate = keyof typeof RELATIVE_DATES
+
+export enum ParameterType {
+  Text = 'text',
+  Document = 'document',
```

Comment:
nitpick: File is shorter and matches with the wording OAI uses for it's type file message

Reply:
mmm, the thing is that we try to convert the document to plain text. If we say 'file' everywhere it seems that I can upload an image or even a video...

Reply:
Also, the new content type from Anthropic does refer to it as document too:
```
"content": [{
            "type": "document",
            "source": {
                "type": "base64",
                "media_type": "application/pdf",
                "data": $PDF_BASE64
            }
        },
```